corehq/apps/dump_reload/sql/dump.py

```diff
@@ -19,6 +19,7 @@
     'form_processor.XFormOperationSQL': SimpleFilter('form__domain'),
     'form_processor.CommCareCaseSQL': SimpleFilter('domain'),
     'form_processor.CommCareCaseIndexSQL': SimpleFilter('domain'),
+    'form_processor.CaseAttachmentSQL': SimpleFilter('case__domain'),
```

Comment:
Do case attachments exist for icds? I thought that case attachments aren't released/used for performance reasons. (agreed it should be here for completeness though)